### PR TITLE
Added Cicular QR Design

### PR DESCRIPTION
**Pull request by hritikaggarwal31**

### CODE REVIEW

File: src/Types.ts [hunk 1]
```diff
@@ -60,6 +60,8 @@ export interface QRDrawingConfig {
     logoMargin: number;
     logoCornerRadius: number;
     maskedDots: boolean;
+    designStyle?:string;
```
Comment (sourobrata) [inline src/Types.ts:63]: @hritikaggarwal31 Are we still using these attributes?

Comment (hritikaggarwal31) [reply src/Types.ts]: No, removed these attributes.

File: src/Svg.ts [hunk 2]
```diff
@@ -1208,7 +1281,7 @@ export class SVGDrawing {
     }
 
     private async drawFrame(canvas: object, frameStyle: QRCodeFrame | undefined, frameColor: string | undefined, frameText: string | undefined) {
-        if (!frameStyle || frameStyle === QRCodeFrame.NONE) {
+        if (!frameStyle || frameStyle === QRCodeFrame.NONE || frameStyle === QRCodeFrame.CIRCULAR) {
```
Comment (sourobrata) [inline src/Svg.ts:1284]: @hritikaggarwal31 Should we have this condition for Models.ts as well?

**Comment by sourobrata** [comment]
@hritikaggarwal31 Please fix the lint issues.

File: src/tests/Models.test.ts [hunk 3]
```diff
@@ -299,6 +299,104 @@ const config8 = {
     useOpacity: false,
 };
 
+const config9 = {
```
Comment (sourobrata) [inline src/tests/Models.test.ts:302]: Please move to QR.tests.ts from Models.test and SVG.test

File: src/Svg.ts [hunk 4]
```diff
@@ -181,12 +185,134 @@ export class SVGDrawing {
...
+        return !(inX && inY);
+    }
+    private checkCircle(x: number, y: number, r: number , cx: number) {
+        if((x-cx)*(x-cx) + (y-cx) * (y-cx) <= r*r) {
+            return true;
+        }
+        return false;
+    }
+    private middleSquare(seed: number) {
+        const result = (seed * seed).toString().padStart(4,"0").slice(1, 3);
```
Comment (sourobrata) [inline src/Svg.ts:220]: Please check for support of padStart in IE

File: src/Svg.ts [hunk 5]
```diff
@@ -181,12 +185,134 @@ export class SVGDrawing {
...
+        let ret = parseInt(result, 10);
```
Comment (sourobrata) [inline src/Svg.ts:221]: Please use a more descriptive variable name instead of 'ret'

**Review by sourobrata — APPROVED** [review]

**Pull request merged**